graylog2-web-interface/src/components/datanode/migrations/in-place/JournalDowntimeWarning.tsx

```diff
@@ -21,21 +21,28 @@ import { Alert } from 'components/bootstrap';
 import type { MigrationStepComponentProps } from 'components/datanode/Types';
 import MigrationStepTriggerButtonToolbar from 'components/datanode/migrations/common/MigrationStepTriggerButtonToolbar';
 
+import useJournalDowntimeSize from '../../hooks/useJournalDowntimeSize';
+
 const DownsizeWarning = styled(Alert)`
   margin-top: 10px;
   margin-bottom: 5px;
 `;
 
-const JournalDowntimeWarning = ({ currentStep, onTriggerStep }: MigrationStepComponentProps) => (
-  <>
-    <h3>Journal downtime size warning</h3>
-    <p>Please note that during migration data processing  will stop on your Graylog node, this will result in the journal growing in size.</p>
-    <p>Therefore increase your journal volume size.</p>
-    <DownsizeWarning bsStyle="danger">
-      <p>Please make sure your journal volume size is sufficient before proceeding.</p>
-    </DownsizeWarning>
-    <MigrationStepTriggerButtonToolbar nextSteps={currentStep.next_steps} onTriggerStep={onTriggerStep} />
-  </>
-);
+const JournalDowntimeWarning = ({ currentStep, onTriggerStep }: MigrationStepComponentProps) => {
+  const { data } = useJournalDowntimeSize();
```

Comment:
We should also handle displaying a message when there is an error when getting the throughput estimation.